flight/Libraries/math/pid.c

```diff
@@ -199,6 +230,67 @@ void pid_configure(struct pid *pid, float p, float i, float d, float iLim)
 }
 
 /**
+ * Assigns a pid_deadband struct to the PID controller.
+ * @param[out] pid The PID structure to configure
+ * @param[in] deadband The external deadband structure to assign
+ */
+void pid_assign_deadband(struct pid *pid, struct pid_deadband *db)
+{
```

Comment:
It was an odd way to make sure it gets used when it should be, because the pointer to the deadband was inside the pid struct. I remember you said that autotune wouldn't like it, thus the desire to disable it. Check the new rebased PR, I'm passing it as a parameter instead, which I guess is more elegant to begin with.